packages/cssnano-preset-default/package.json

```diff
@@ -12,36 +12,36 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "css-declaration-sorter": "^3.0.0",
-    "cssnano-util-raw-cache": "^4.0.0",
+    "css-declaration-sorter": "^4.0.1",
+    "cssnano-util-raw-cache": "^4.0.1",
     "postcss": "^7.0.0",
-    "postcss-calc": "^6.0.0",
+    "postcss-calc": "^6.0.2",
     "postcss-colormin": "^4.0.0",
-    "postcss-convert-values": "^4.0.0",
-    "postcss-discard-comments": "^4.0.0",
-    "postcss-discard-duplicates": "^4.0.0",
-    "postcss-discard-empty": "^4.0.0",
-    "postcss-discard-overridden": "^4.0.0",
-    "postcss-merge-longhand": "^4.0.0",
-    "postcss-merge-rules": "^4.0.0",
-    "postcss-minify-font-values": "^4.0.0",
-    "postcss-minify-gradients": "^4.0.0",
-    "postcss-minify-params": "^4.0.0",
-    "postcss-minify-selectors": "^4.0.0",
-    "postcss-normalize-charset": "^4.0.0",
-    "postcss-normalize-display-values": "^4.0.0",
-    "postcss-normalize-positions": "^4.0.0",
-    "postcss-normalize-repeat-style": "^4.0.0",
-    "postcss-normalize-string": "^4.0.0",
-    "postcss-normalize-timing-functions": "^4.0.0",
-    "postcss-normalize-unicode": "^4.0.0",
-    "postcss-normalize-url": "^4.0.0",
-    "postcss-normalize-whitespace": "^4.0.0",
-    "postcss-ordered-values": "^4.0.0",
-    "postcss-reduce-initial": "^4.0.0",
-    "postcss-reduce-transforms": "^4.0.0",
-    "postcss-svgo": "^4.0.0",
-    "postcss-unique-selectors": "^4.0.0"
+    "postcss-convert-values": "^4.0.1",
+    "postcss-discard-comments": "^4.0.1",
+    "postcss-discard-duplicates": "^4.0.2",
+    "postcss-discard-empty": "^4.0.1",
+    "postcss-discard-overridden": "^4.0.1",
+    "postcss-merge-longhand": "^4.0.6",
+    "postcss-merge-rules": "^4.0.2",
+    "postcss-minify-font-values": "^4.0.2",
+    "postcss-minify-gradients": "^4.0.1",
+    "postcss-minify-params": "^4.0.1",
+    "postcss-minify-selectors": "^4.0.1",
+    "postcss-normalize-charset": "^4.0.1",
+    "postcss-normalize-display-values": "^4.0.1",
+    "postcss-normalize-positions": "^4.0.1",
+    "postcss-normalize-repeat-style": "^4.0.1",
+    "postcss-normalize-string": "^4.0.1",
+    "postcss-normalize-timing-functions": "^4.0.1",
+    "postcss-normalize-unicode": "^4.0.1",
+    "postcss-normalize-url": "^4.0.1",
+    "postcss-normalize-whitespace": "^4.0.1",
+    "postcss-ordered-values": "^4.1.1",
+    "postcss-reduce-initial": "^4.0.2",
+    "postcss-reduce-transforms": "^4.0.1",
+    "postcss-svgo": "^4.0.1",
+    "postcss-unique-selectors": "^4.0.1"
```

Comment:
No need update other plugin

Reply:
version 4.0.0 has in `package.json` `postcss@6` but version 4.0.1 defines version 7.0.
it is reason why minimal version should be 4.0.1 or 4.0.2

Reply:
:+1: